content_script.js

```diff
@@ -1,39 +1,20 @@
 const isDebugMode = true;
 log("Plugin start");
 var json = {}
-const currentURL = new URL(document.URL);
-const pathArray = currentURL.pathname.split('/');
-const baseURL = "https://api.swift.gg/content/";
```

Comment:
Why do we move this line into a function? I think if we have muti calls for this plugin, we may need a baseURL to a further extent.

Reply:
because the url may change and the script may not recalled. So we made after function. And every time we need it, we just call it. So we can always get a correct url.